### PR TITLE
Hut8/reprovision primary

### PR DESCRIPTION
**Pull request by hut8**

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -46,8 +46,7 @@
 
 task :restart_workers do
   on roles(:worker) do
-    sudo 'stop workers || true'
-    sudo 'start workers'
+    sudo 'systemctl restart rails-assets-build.service'
```
Comment (joshjordan) [inline config/deploy.rb]: Does `systemctl restart` cause the workers to start if they aren't already running? That was the purpose of the previous script.

File: config/deploy/production.rb [hunk 2]
```diff
@@ -1,17 +1,11 @@
 server(
-  'srv.rails-assets.org',
+  'therac25.rails-assets.org',
```
Comment (joshjordan) [inline config/deploy/production.rb:2]: Would love to find a better way to share these with ansible easily, someday.

File: config/deploy/production.rb [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
-  user: fetch(:deploy_user),
-  roles: %w(worker)
-)
-
-server(
-  'wopr.rails-assets.org',
-  user: fetch(:deploy_user),
-  roles: %w(app db web)
-)
+# server(
```
Comment (hut8) [inline config/deploy/production.rb:7]: I have to reprovision wopr too since it's 14.04

File: infrastructure/lets-encrypt-renew-certificate [hunk 4]
```diff
@@ -1,36 +1,50 @@
-#!/bin/bash
-
-web_service='nginx'
+#!/usr/bin/zsh
```
Comment (hut8) [inline infrastructure/lets-encrypt-renew-certificate:1]: oops this file is unnecessary now.

**Comment by joshjordan** [comment]
👍

**Pull request merged**